backend/src/server/app.ts

```diff
@@ -39,6 +42,7 @@ type TMain = {
 // Run the server!
 export const main = async ({ db, smtp, logger, queue, keyStore }: TMain) => {
   const appCfg = getConfig();
+  const cronJobs: CronJob[] = [];
```

Comment:
Hmm i need to think more about this pattern. Feels kinda very weird singleton pattern.

You can start the cron jobs in `routes/index` like normal and then do a server.register for cleanup event. I ideally i am thinking what if we move rate limiting logic inside the `routes` section as its not part of app logic

Reply:
moved the cron setup in the routes section